### DISCUSSION TITLE
What do you think of Jetpack Compose in this project?

### DISCUSSION THREAD

**Discussion by inson1** [clementwzk/OpenCalc#376] [Ideas]
In future ofc. After everything is done.

**Comment by clementwzk** [comment]
No idea, I've never used it. I don't really see what it would change

**Reply by inson1** [reply]
Its should be easier to mantain, new faster development and its "the future". Its replacing xml for buillding GUI in Kotlin/Android.
https://developer.android.com/jetpack/compose/why-adopt
